mobile-ui/build.gradle

```diff
@@ -86,6 +86,8 @@ android {
 
 kapt {
     correctErrorTypes = true
+    useBuildCache = true  // Gradle version 4.3 and above is required
```

Comment:
You are using 4.3??

Reply:
I'm not in 4.3, it doesn't work now. I just put it in temporally. Once our gradle version upgrades to 4.3 we can use it directly.